client/src/www/app/main.cordova.ts

```diff
@@ -76,7 +76,12 @@ class CordovaErrorReporter extends SentryErrorReporter {
 
 class CordovaMethodChannel implements MethodChannel {
   invokeMethod(methodName: string, params: string): Promise<string> {
-    return pluginExecWithErrorCode('invokeMethod', methodName, params);
+    try {
+      return pluginExecWithErrorCode('invokeMethod', methodName, params);
+    } catch (e) {
+      console.debug('invokeMethod failed', methodName, e);
```

Comment:
error instead of debug? Also, are you planning to catch the error of "initiating" the promise or "during" the promise? This will decide whether we need to add `await` here.

```suggestion
      console.error('invokeMethod failed', methodName, e);
```

Reply:
I don't think this should be errors, because the calls may fail for various reasons (like failed to parse the config in the key validation). This is really for debugging. The caller should decide whether to log.